Pod/Classes/SEGUrbanAirshipIntegration.m

```diff
@@ -58,9 +58,11 @@ - (instancetype)initWithSettings:(NSDictionary *)settings {
             config.developmentAppKey = settings[kUrbanAirshipAppKey];
             config.developmentAppSecret = settings[kUrbanAirshipAppSecret];
         }
-
-        // Call takeOff (which creates the UAirship singleton)
-        [UAirship takeOff:config];
+        // Call takeOff on main thread (which creates the UAirship singleton)
+       dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
It works to remove the issue. Are there some specific failure modes we are looking for?